pkgs/by-name/jr/jreleaser-cli/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchurl, makeWrapper, jre }:
+stdenv.mkDerivation rec {
+  pname = "jreleaser-cli";
+  version = "1.11.0";
+
+  src = fetchurl {
+    url = "https://github.com/jreleaser/jreleaser/releases/download/v${version}/jreleaser-tool-provider-${version}.jar";
+    sha256 = "sha256-VkINXKVBBBK6/PIRPMVKZGY9afE7mAsqrcFPh2Algqk=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/share/java/ $out/bin/
+    cp $src $out/share/java/${pname}.jar
+    makeWrapper ${jre}/bin/java $out/bin/${pname} \
+      --add-flags "-jar $out/share/java/${pname}.jar"
+  '';
+
+  meta = with lib; {
```

Comment:
the usage of `with lib;` should be reduced so please use lib.sourceTypes, lib.licenses and lib.maintainers instead